src/indicators.ts

```diff
@@ -59,19 +60,26 @@ export class Indicators {
         return new Promise<Section>((resolve, reject) => {
             this.xmlParse.parseContent(xmlFile, (err, data) => {
                 if (err) { return reject(err); }
-                const section = data.find((lcovSection) => {
-                    // consider windows and linux file paths
-                    let cleanFile = file.replace(/[\\\/]/g, "");
-                    let cleanLcovFileSection = lcovSection.file.replace(/[\\\/]/g, "");
-
-                    // on Windows remove drive letter from path because of cobertura format
-                    // also convert both path to lowercase because Windows's filesystem is case insensitive
-                    if (process.platform === "win32") {
-                        cleanFile = cleanFile.substr(2).toLowerCase();
-                        cleanLcovFileSection = cleanLcovFileSection.toLowerCase();
-                    }
 
-                    return cleanFile.includes(cleanLcovFileSection);
+                const cleanFileToMatch = file.split(/[\\\/]/).reverse();
+
+                // Find best path match
+                let bestMatchLength = 0;
+                let section = null;
+
+                _.forEach(data, (xmlSection) => {
```

Comment:
I walked over this section with the debugger and it performances nicely. It also looks very clean code wise, nice work!